hw/ip/otp_ctrl/data/otp_ctrl.hjson

```diff
@@ -35,41 +32,40 @@
   // Parameters //
   ////////////////
   param_list: [
-    { name: "NumLcPartitionWords",
-      desc: "Number of 32bit words in the life cycle partition",
+    { name: "OtpByteAddressWidth",
+      desc: "Width of the OTP Byte address.",
```

Comment:
Looks like some desc has a period at the end (like this one), and some does not (line 42). Maybe we can align them all?

Reply:
Sure, let's align those.